plutus-core/executables/Common.hs

```diff
@@ -282,10 +286,10 @@ parseInput ::
 parseInput inp = do
     contents <- getInput inp
     -- parse the UPLC program
-    case parseProgram contents of
+    case parseProgram (show inp) contents of
       -- when fail, pretty print the parse errors.
       Left (err :: ParserErrorBundle) ->
-        errorWithoutStackTrace $ show err
+          errorWithoutStackTrace $ PP.render $ pretty err
```

Comment:
This is the fix for the main problem.

Reply:
I've pushed a commit that uses `errorBundlePretty` instead. It may be better for in case we do configure how we pretty print the parse errors later. It's up to you whether you take the commit or not, I'm fine either way. You can also use `display` here if you want to do this way.

Reply:
Ah, thanks.  I get lost in all these different ways of printing things.  I just copied what it said a few lines below, but that's for a completely different kind of error.